src/applications/personalization/profile/tests/components/direct-deposit/DirectDeposit.unit.spec.jsx

```diff
@@ -237,4 +236,39 @@ describe('authenticated experience -- profile -- unified direct deposit', () =>
       expect(getByRole('button', { name: /save/i })).to.exist;
     });
   });
+  describe('Montgomery GI Bill', () => {
+    it('Renders GI Bill additional information', () => {
+      const triggerText =
+        'How to update your direct deposit information for Montgomery GI Bill';
+      const additionalInfoText =
```

Comment:
nit: maybe we should use a `data-test-id` here instead of the content text, so that we don't need to change the tests if the content ever needs to change.